src/Converters/PollyConverter.php

```diff
@@ -54,18 +66,40 @@ public function getClient(): \Aws\Polly\PollyClient
      */
     public function convert($data, array $options = null)
```

Comment:
I made the `convert` method more readable and move the checking for the voice ID and output format in its own method.